modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js

```diff
@@ -583,7 +583,7 @@ AUI.add(
 						var defaultLanguageId = instance.get('defaultLanguageId');
 						var localizedName = instance.get('localizedName');
 
-						if (!localizedName[defaultLanguageId]) {
+						if ((!localizedName[defaultLanguageId]) || (!localizedName[defaultLanguageId].trim())) {
```

Comment:
hey @marcelabc , can't we just leave `(!localizedName[defaultLanguageId].trim())` instead of both conditions? It seems redundant to me. Can you check that? :)